docs/src/main/asciidoc/security-getting-started-tutorial.adoc

```diff
@@ -415,7 +419,7 @@ In this scenario, `Dev Services for PostgreSQL` launches and configures a `Postg
 %prod.quarkus.datasource.db-kind=postgresql
 %prod.quarkus.datasource.username=quarkus
 %prod.quarkus.datasource.password=quarkus
-%prod.quarkus.datasource.jdbc.url=jdbc:postgresql:elytron_security_jpa
+%prod.quarkus.datasource.jdbc.url=jdbc:postgresql://localhost/quarkus
```

Comment:
@jedla97 Can you clarify this update please

Reply:
Main motivation for me is that this should be same as quickstart as quickstart is present as solution. For source see https://github.com/quarkusio/quarkus-quickstarts/blob/main/security-jpa-quickstart/src/main/resources/application.properties#L4